src/nifgen/templates/functions.rst/get_fir_filter_coefficients.rst.mako

```diff
@@ -0,0 +1,42 @@
+<%page args="function, config, method_template, indent"/>\

```

Comment:
Are you keeping these on purpose because they may be useful in the future?
I'm not sure what the proper way of handling this sort of thing is.

Reply:
Yes, I am keeping this on purpose. I mentioned that in the PR description. I don't want to lose this knowledge since it isn't obvious how this needs to work.

Reply:
It just seems like dead code so there's a risk of it being removed in the future too.

How about removing it but in the issue recording the last commit that contained it to show a solution?